### DISCUSSION TITLE
My YOLO achievement is wrong and I would like it removed.

### DISCUSSION THREAD

**Discussion by iwillig** [community/community#28755] [Other Feature Feedback, Questions, & Ideas]
I never merge PR's without another person's approval. The PR that you all claim is my yolo merge was actually approved and merged in the same minute. I believe your process for calculating that achievement does not consider seconds. 

Thanks.

**Comment by portothree** [comment]
YOLO

**Comment by arakshit-tibco** [comment]
same with me - https://github.com/community/community/discussions/28806

**Comment by radar-blip** [comment]
The option to remove just 1 achievement badge doesn't exist at the moment. You can only disable all of them (in profile settings).
